corehq/apps/hqwebapp/fields.py

```diff
@@ -25,20 +24,14 @@ def prepare_value(self, value):
 class MultiCharField(forms.Field):
     """
     A text field that expects a comma-separated list of inputs, and by default
-    uses the AutocompleteTextarea widget, which uses a jQuery plugin to provide
-    autocompletion (depends on Bootstrap) based on the 'choices' constructor
-    argument.
-
+    uses select2 widget that allows for multiple selections and accepts free text.
```

Comment:
I feel like this class could be further simplified, combined with `MultiEmailField`, or possibly eliminated altogether, but I've been staring at it for too long. Open to suggestions.